clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala

```diff
@@ -69,22 +80,62 @@ class Exporter(spark : SparkSession, apiClient: ApiClient, repoName: String, dst
       WHERE n.etag <> p.etag OR n.etag is null or p.etag is null)
     """)
 
-    export(ns, dst, actionsDF, false)
-    export(ns, dst, actionsDF, true)
+    actOnActions(ns, dst, commitID, actionsDF)
+  }
+
+  final val successMsg = "Export completed successfully!"
+  private def actOnActions(ns: String, dst: String, commitID: String, actionsDF: DataFrame): Unit = {
+    if (!export(ns, dst, commitID, actionsDF, false)) {
+      // failed to export files, don't export _success files
+      return
+    }
+    if (!export(ns, dst, commitID, actionsDF, true)){
+      return
+    }
+
+    writeSummaryFile(true, commitID, successMsg)
+  }
+
+
+
+  private def writeSummaryFile(success: Boolean, commitID: String, content : String) = {
+    val suffix = if(success) "SUCCESS" else "FAILURE"
+    val dstPath = resolveURL(new URL(dstRoot), s"EXPORT_${commitID}_${suffix}")
+    val dstFS = dstPath.getFileSystem(spark.sparkContext.hadoopConfiguration)
+
+    val stream = dstFS.create(dstPath)
+    stream.writeChars(content)
+    stream.close()
+  }
+
+  final val prefixLen = 8
+  private def randPrefix() : String = {
+    val gen = Random.alphanumeric.dropWhile(_.isDigit)
+    gen.take(prefixLen).mkString("")
   }
 }
 
 object Exporter {
-  val sparkSuccessFileSuffix = "_SUCCESS"
+  // spark success files must end with "/_SUCCESS", unless they are in the root
+  // dir, then they should be equal to "_SUCCESS".
+  final val sparkSuccessFileRegex = "((.*\\/_SUCCESS)|(^_SUCCESS))$".r
```

Comment:
```suggestion
  final val sparkSuccessFileRegex = "((.*/_SUCCESS)|(^_SUCCESS))$".r
```
or
```suggestion
  final val sparkSuccessFileRegex = raw"((.*/_SUCCESS)|(^_SUCCESS))$".r
```
(see https://www.scala-lang.org/api/current/scala/util/matching/Regex.html)
or improve the regex,
```suggestion
  final val sparkSuccessFileRegex = "(^|/)_SUCCESS$".r
```